game/games/pixel_side_scroller/pawns/character/character.tscn

```diff
@@ -75,7 +75,7 @@ tracks/0/keys = {
 }
 
 [sub_resource type="RectangleShape2D" id=4]
-extents = Vector2( 4, 7.5 )
+extents = Vector2( 3.94, 7.5 )
```

Comment:
I assume this change was done so the character properly fits through narrow 1 tile wide gaps?